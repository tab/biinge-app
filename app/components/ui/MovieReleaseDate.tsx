import React from "react"

import { formatDate } from "helpers/formatDate"
import Typography from "components/ui/Typography"
import { movieStyles } from "styles"

type Props = {
  children: string
}

const MovieReleaseDateComponent = ({ children }: Props) => {
  return (
    <Typography variant="headline" style={movieStyles.date}>
      {formatDate(children)}
    </Typography>
  )
}

export default MovieReleaseDateComponent
