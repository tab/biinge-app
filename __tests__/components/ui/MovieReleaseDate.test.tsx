import React from "react"
import "react-native"
import { render } from "@testing-library/react-native"

import { it } from "@jest/globals"

import MovieReleaseDate from "components/ui/MovieReleaseDate"

describe("MovieReleaseDate", () => {
  const release_date = "2024-02-27"

  it("renders correctly", () => {
    const tree = render(<MovieReleaseDate>{release_date}</MovieReleaseDate>)
    expect(tree).toMatchSnapshot()
  })
})
