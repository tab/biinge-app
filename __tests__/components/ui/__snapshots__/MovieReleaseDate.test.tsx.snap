// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReleaseDate renders correctly 1`] = `
<Text
  ellipsizeMode="tail"
  style={
    [
      {
        "color": "#181818",
        "fontSize": 17,
        "fontWeight": "600",
        "lineHeight": 22,
      },
      {
        "color": "#666666",
      },
    ]
  }
>
  27.02.2024
</Text>
`;
